store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/IngestUtils.java

```diff
@@ -81,12 +81,6 @@ public static int createSplitsFile(final Connector conn, final String table, fin
         }
         // This should have returned at most maxSplits splits, but this is not implemented properly in MockInstance.
         if (splits.size() > maxSplits) {
-            if (conn instanceof MockConnector) {
-                LOGGER.info("Manually reducing the number of splits to {} due to MockInstance not implementing"
-                        + " listSplits(table, maxSplits) properly", maxSplits);
-            } else {
-                LOGGER.info("Manually reducing the number of splits to {} (number of splits was {})", maxSplits, splits.size());
```

Comment:
Is it not worth logging this anymore? As we are never using a `MockConnector`?

Reply:
I copied this from the old PR for upgrading to Accumulo 2. Accumulo 2 doesn't have `MockConnector` or a direct equivalent so this needed to be dropped.

Reply:
I meant the else part, as it effectively should always be hitting that else block now

Reply:
I simply copied this change over from the old PR, without taking too much of a look at it. You're right that this doesn't make much sense. Depending on what happened to `MockInstance` it might be that more code should be removed. As the comment implies the situation of `splits.size() > maxSplits` is related to `MockInstance`. If there's no `MockInstance` in use anymore then the whole `if` could potentially be removed. I'll take a further look at this.